### PR TITLE
Kim/goth 314

### PR DESCRIPTION
**Pull request by kimlarocca**
changed the home page featured article to pull from the first story in the home page content collection (per Sam)

https://nypublicradio-digital.atlassian.net/browse/GOTH-314

### CODE REVIEW

File: composables/data/articlePages.ts [hunk 1]
```diff
@@ -108,4 +108,51 @@ export function normalizeArticlePage(article: Record<string, any>): ArticlePage
 // Transform a list of article page data from the API into a simpler and typed format
 export function normalizeFindArticlePagesResponse(articlesResponse: any): ArticlePage[] {
     return articlesResponse.value.items?.map(normalizeArticlePage)
+}
+
+// Transform the featured article page data from the API into a simpler and typed format
+// The home page featured article should display only the first story in the content collection
+export function normalizeHomePageFeaturedArticle(article: Record<string, any>): ArticlePage {
```
Comment (walsh9) [inline composables/data/articlePages.ts:115]: I don't think we need to duplicate this whole function just to call it on a deeper nested value, we can just call `normalizeArticlePage(article.value.pageCollectionRelationship?.[0].pages?.[0])`
Unless I'm misunderstanding what this is doing.

File: pages/index.vue [hunk 2]
```diff
@@ -3,7 +3,11 @@ import VCard from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VCar
 import VByline from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VByline.vue'
 import useImageUrl from '~~/composables/useImageUrl'
 
-const featuredArticles = await findFeaturedArticlePages('').then(({ data }) =>
+const featuredArticle = await findPage('/').then(({ data }) =>
+  normalizeHomePageFeaturedArticle(data)
```
Comment (walsh9) [inline pages/index.vue:7]: for example:
```
const featuredArticle = value.pageCollectionRelationship?.[0].pages?.[0]
return featuredArticle && normalizeArticle(featuredArticle)
```

**Review by walsh9 — CHANGES_REQUESTED** [review]

File: composables/data/articlePages.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@ import { ArticlePage } from "../types/Page"
 import Author from '../types/Author'
 
 // get the 4 featured article pages
-export async function findFeaturedArticlePages(queryParams: any) {
+export async function findLatestArticlePages(queryParams: any) {
```
[the remark this answers is not in the document]
Comment (kimlarocca) [reply composables/data/articlePages.ts:5]: That's better, thanks! I updated this.

File: composables/data/articlePages.ts (hunk 1 above)
Comment (kimlarocca) [reply composables/data/articlePages.ts:115]: Another good idea :)

File: pages/index.vue (hunk 2 above)
Comment (kimlarocca) [reply pages/index.vue:7]: Gotcha - I did

`const featuredArticle = await findPage('/').then(({ data }) =>
  normalizeArticlePage(data.value.pageCollectionRelationship?.[0].pages?.[0])
)`

I thought you wanted a new function to handle this, but you actually said you just wanted a new type to handle the home.HomePage - but I ended up not needing this

**Review by walsh9 — APPROVED** [review]

**Pull request merged**